### PR TITLE
Minor styling fixes

### LINKED ISSUE #1127: collaborator modal top gets cut off on my 13" screen
screen shot 2018-03-07 at 6 31 31 pm

### PR DESCRIPTION
**Pull request by broskoski**
Closes #1127 

collaborator

### CODE REVIEW

File: react/components/UI/ModalDialog/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Dialog = styled.div`
   width: 90%;
   height: 90%;
   max-width: 40em;
-  max-height: 60em;
+  max-height: calc(100vh - 7em);
```
Comment (dzucconi) [inline react/components/UI/ModalDialog/index.js]: This is gonna look odd on super tall screens. If this is a workaround for Safari there's a better way (I don't know off hand)

File: react/components/UI/Modal/Modal.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const ModalBackdrop = styled.div`
   left: 0;
   background-color: rgba(255, 255, 255, 0.9);
   z-index: 6001;
+  transform: translate3d(0, 0, 1px);
```
Comment (dzucconi) [inline react/components/UI/Modal/Modal.js]: ?

Comment (dzucconi) [reply react/components/UI/Modal/Modal.js]: If you're trying to force GPU then use `transform: translatez(0);` —this is almost certainly going to cause aliasing bugs.

Comment (broskoski) [reply react/components/UI/Modal/Modal.js]: Moves it up one pixel on the z-axis

Comment (dzucconi) [reply react/components/UI/Modal/Modal.js]: Also why

File: react/components/UI/ModalDialog/index.js (hunk 1 above)
Comment (dzucconi) [reply react/components/UI/ModalDialog/index.js]: Fine for now though. But I'd leave a note about the underlying issue: Safari flexbox blah blah

File: react/components/UI/Modal/Modal.js (hunk 2 above)
Comment (dzucconi) [reply react/components/UI/Modal/Modal.js]: I know what it does but that doesn't explain anything.

Comment (broskoski) [reply react/components/UI/Modal/Modal.js]: We do this on the header bar to fix position:fixed bugs on mobile safari

Comment (dzucconi) [reply react/components/UI/Modal/Modal.js]: Did you actually experience an issue on Mobile Safari? Can you document why it needs to be 1px in a comment pointing to some external source? Can we move it into a media query that only executes on mobile devices.

**Pull request merged**